### PR TITLE
Python3 6 version

### PR DESCRIPTION
**Pull request by Paulokim1**
[ ] Wrote test for feature
[x] Added changes to CHANGELOG.md

**Changes proposed:**
Modified setup.py due to 3.0 limitations. Also, a simpler approach to use versions with use of import, taking out the need of using the with open commad and iterate through a file.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,9 @@ All notable changes to this project will be documented in this file.
 
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+### v0.17.6
```
Comment (adnanhemani) [inline CHANGELOG.md:7]: Hi @Paulokim1 - thanks for this. As per https://github.com/data-8/datascience/issues/559, can we please increment the package version to 0.18.0? Thanks!

**Review by adnanhemani — CHANGES_REQUESTED** [review]

File: setup.py [hunk 2]
```diff
@@ -46,7 +40,8 @@ def run_tests(self):
 setup(
     name = 'datascience',
     packages = ['datascience'],
-    version = version,
+    python_requires='>=3.6',
+    version = __version__,
```
Comment (davidwagner) [inline setup.py:44]: Why change `version = version` to `version = __version__`?  This seems like it changes the logic of this code.  What is the purpose of this?

**Review by davidwagner — CHANGES_REQUESTED** [review]
We need to update datascience/version.py to set the version to 0.18.0, to match the updated CHANGELOG

**Review by adnanhemani — APPROVED** [review]
@davidwagner - just fixed this CR up. Let me know if this works :)

File: setup.py (hunk 2 above)
Comment (adnanhemani) [reply setup.py]: This makes the build fail due to a circular dependency. I've reverted this logic now.

**Review by davidwagner — APPROVED** [review]

**Pull request merged**